### PR TITLE
add check-upgrade.sh to sensu-plugins-routeros

### PR DESCRIPTION
**Pull request by x70b1**
Pull Request contains the Script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,38 @@
-# Sensu Plugin Skeleton
+## Sensu-Plugins-skel
```
Comment (majormoses) [inline README.md]: Should update with the actual plugin name this was just a copy/paste left over from generating this repo from our skel.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
 
-All the files you'll need to start your own plugin written in Ruby [for Sensu](https://github.com/sensu/sensu), the monitoring framework.
+[![Build Status](https://travis-ci.org/sensu-plugins/sensu-plugins-skel.svg?branch=master)](https://travis-ci.org/sensu-plugins/sensu-plugins-skel)
```
Comment (majormoses) [inline README.md]: lets update the plugin name

File: README.md [hunk 3]
```diff
@@ -1,24 +1,38 @@
...
+[![Gem Version](https://badge.fury.io/rb/sensu-plugins-skel.svg)](http://badge.fury.io/rb/sensu-plugins-skel)
```
Comment (majormoses) [inline README.md]: lets update the plugin name

File: README.md [hunk 4]
```diff
@@ -1,24 +1,38 @@
...
+[![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-skel.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-skel)
```
Comment (majormoses) [inline README.md]: lets update the plugin name

File: README.md [hunk 5]
```diff
@@ -1,24 +1,38 @@
...
-- [ ] If you end up loving maintaining your plugin, [volunteer to be an org maintainer](https://github.com/sensu-plugins/community#how-you-can-help)
+The default values for SNMP are:
+* PORT=161
+* COMMUNITY=public
+
+For a check over SSH:
+* PORT=22
+* USER=user
+
+You need an user and SSH keys. There is no option for a password.
```
Comment (majormoses) [inline README.md:25]: :+1:

File: bin/check-upgrade.sh [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        "bugfix")
+            param_version="6fix"
+            ;;
+        *)
+            param_version="6"
+            ;;
+    esac
+
+    # check the MikroTik server for upgrades
+    if ! routeros_available=$(curl -fsA "check_routeros-upgrade" https://download.mikrotik.com/routeros/LATEST.$param_version); then
```
Comment (majormoses) [inline bin/check-upgrade.sh]: just a thought on enhancing this to allow it to be an argument for the url as someone might need to host their own in airgapped setups. Not required but would be nice to have.

**Review by majormoses — CHANGES_REQUESTED** [review]
Overall looks good but a few things to fixup and some optional improvements.

File: bin/check-upgrade.sh (hunk 6 above)
Comment (x70b1) [reply bin/check-upgrade.sh]: You mean something like proxy vars?

**Comment by x70b1** [comment]
Can you replace routeros with `RouterOS` in the repo description?
Just to be accurate.

File: bin/check-upgrade.sh (hunk 6 above)
Comment (majormoses) [reply bin/check-upgrade.sh]: no I mean maybe I am required to not allow external traffic even through a proxy. For proxies you can use the ENV vars and curl will honor it. What I mean is that maybe I want to host the content on:  https://download.domain.tld/routeros/LATEST.$param_version which could be on an internal server

Comment (x70b1) [reply bin/check-upgrade.sh]: I added an ENV var. Now you can replace the default server if you want.

**Comment by x70b1** [comment]
I added `ROUTEROS_UPDATEURL` so that you can redefine the URL. 
Is there anything more I can do?

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
sorry it slipped through the cracks, cutting a release. Thanks for the awesome work.

**Comment by x70b1** [comment]
Thank for yours too!

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-routeros/versions/0.1.0

**Pull request merged**